Classification/test/Classification/deprecated_test_classification_point_set.cpp

```diff
@@ -77,7 +77,7 @@ int main (int, char**)
                                color_map, echo_map);
 
   CGAL_assertion (generator.number_of_scales() == 5);
-  CGAL_assertion (features.size() == 80);
+  CGAL_assertion (features.size() == 44);
```

Comment:
So that test was wrong. All those green cells in the [test suite results][1] shows that we have a problem:
* several platforms do not have the pre-requisites but the cells are not blue: the sentence "NOTICE: Boost Serialization or IO Streams not found, no test will be compiled." does not match the regular expression. See [the section "Test suite results" of the guidelines][2],
* on "Release" platforms: the code of tests must use `assert`, instead of `CGAL_assertion`, so that "Release" platforms, with `-DCGAL_NDEBUG` (but not `-DNDEBUG`) can still evaluate assertions of tests. `CGAL_assertion` is only for the code of header files. so that our users can disable them. See [the guidelines][3].

[1]: https://cgal.geometryfactory.com/CGAL/testsuite/results-4.13-Ic-39.shtml#Classification
[2]: https://github.com/CGAL/cgal/wiki/Testing#test-suite-results
[3]: https://github.com/CGAL/cgal/wiki/Testing#introduction